rfc/009-distributed-coordinator.md

```diff
@@ -0,0 +1,178 @@
+# RFC 009: Distributed coordinator
+
+## Background
+
+The Contrast Coordinator is a stateful service with a backend storage that can't be shared.
```

Comment:
Is the coordinator state non-sensitive?

Reply:
It does not need to be CC-secure, if that's what you mean. https://github.com/edgelesssys/contrast/blob/d4892d3258737c4c69be8ce3443760ffc4ac85b0/rfc/004-recovery.md?plain=1#L38-L40